docs/community/index.rst

```diff
@@ -0,0 +1,37 @@
+=========
+Community
+=========
+
+Pyfar is completely open source and it’s license allows unrestricted
+educational and commercial use. It is made by the acoustics research
+community for the acoustics research community. We welcome any contributions
+– go ahead if you are in the mood.
+
+.. grid:: 1 1 2 2
+   :gutter: 4
+
+   .. grid-item-card::
+      :text-align: justify
+
+      **Feedback, bugs, feature requests**
+      ^^^^
+      The best way for any feedback, bug reports, and feature request is to
+      open a new issue on GitHub. The GitHub projects are linked through the
+      Code buttons on `pyfar.org <https://pyfar.org>`_.
+
+   .. grid-item-card::
+      :text-align: justify
+
+      **Add code, documentation, or funding**
+      ^^^^
+
+      If you want to add code or documentation, a good starting point for ideas
+      are the issues on GitHub. The GitHub projects are linked through the
+      Code buttons `pyfar.org <https://pyfar.org>`_. If you have own ideas for contributions, may it be
```

Comment:
Can we emphasise the word "Code" in any way so that it is clear that this is what the button says?